frontend/src/routes/profile/+page.svelte

```diff
@@ -1,21 +1,324 @@
 
   let user: User
+  let newListName: string
+
+  // TODO: Type theese
+  let favoriteList: any
+  let watchList: any
+  let customLists: any
 
   $: if ($auth) {
     user = parseJwt($auth)
   }
+
+  const getList = async (listType: string) => {
+    const res = await fetch(PYTHON_API + "/" + listType + "?encoded_jwt=" + $auth)
+
+    const json = await res.json()
+
+    if (res.status == 498) {
+      $auth = ""
+    } else if (res.ok) {
+      if (listType == "favorites") {
+        favoriteList = json
+      } else if (listType == "watch_list") {
+        watchList = json
+      } else if (listType == "custom_lists") {
+        customLists = json
+      }
+    }
+    return json
+  }
+
+  const deleteCustomList = async (listId: string) => {
+    const res = await fetch(
+      PYTHON_API + "/custom_lists" + "?id=" + listId + "&encoded_jwt=" + $auth,
+      {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+    if (res.status == 498) {
+      $auth = ""
+    }
+    return await res.json()
+  }
+
+  const createCustomList = async (listName: string) => {
+    const res = await fetch(
+      PYTHON_API + "/custom_lists" + "?list_name=" + listName + "&encoded_jwt=" + $auth,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+
+    const json = await res.json()
+
+    if (res.status == 498) {
+      $auth = ""
+    }
+    return json
+  }
+
+  const getMedia = async (idArray: any[], listType: string, listName: string = "") => {
+    let idString = ""
+
+    for (let i = 0; i < idArray.length; i++) {
+      idString += "&ids=" + idArray[i].streamchaser_id
+    }
+
+    const res = await fetch(PYTHON_API + "/media?c=" + $currentCountry + idString)
+    const media = await res.json()
+
+    if (res.status == 498) {
+      $auth = ""
+    } else if (res.ok) {
+      if (listType == "favorites") {
+        favoriteList.favorites = media.hits
+      } else if (listType == "watch_list") {
+        watchList.watch_list = media.hits
+      } else if (listType == "custom_lists") {
+        for (let i = 0; i < customLists.custom_lists.length; i++) {
+          if (listName == customLists.custom_lists[i].name) {
+            customLists.custom_lists[i].media = media.hits
+          }
+        }
+      }
+    }
+
+    return media.hits
+  }
+
+  //TODO: This is temp data mocking what backend will return
+  const providers = [
+    {
+      provider_name: "Acorn TV",
+      display_priority: 16,
+      provider_id: 87,
+      logo_path: "/5P99DkK1jVs95KcE8bYG9MBtGQ.jpg",
+    },
+    {
+      provider_name: "C More",
+      display_priority: 5,
+      provider_id: 77,
+      logo_path: "/pCIkSBek0aZfPQzOn9gfazuYaLV.jpg",
+    },
+    {
+      provider_name: "Netflix",
+      display_priority: 0,
+      provider_id: 8,
+      logo_path: "/t2yyOv40HZeVlLjYsCsPHnWLk4W.jpg",
+    },
+  ]
```

Comment:
Just adding this so we remember to clean up

Reply:
it has been replaced with real data